lib/helpers/index.js

```diff
@@ -8,6 +10,13 @@ exports.buildQueryRequest = ({ skip, limit, sort, projection, find }) => {
     result.options.skip = skip || 0
     result.options.limit = limit || 100
     if (find) result.query = JSON.parse(find)
+    // URL encoded _id is converted to ObjectId as it will be sent as string
+    if (result?.query?._id && result.query._id.$in) {
```

Comment:
Can `result` be `undefined` here? It doesn't look like it as it's been set above. The same goes with `query`. I believe it's redundant to use `?`

Reply:
Thanks. I updated the code already for this.